database/models/index.js

```diff
@@ -32,5 +32,15 @@ Object.keys(db).forEach(modelName => {
 
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
+/*
+sequelize
+    .authenticate()
+    .then(() => {
+    console.log('Connection has been established successfully.');
+    })
 
+    .catch(err => {
+      console.error('Unable to connect to the database:', err);
+  });
+*/
```

Comment:
Could you remove this commented out code? I have now added a unit tests that checks the DB connection so this is not required anymore.